src/cascadia/TerminalSettingsEditor/ViewModelHelpers.h

```diff
@@ -66,7 +66,7 @@ public:                                                   \
             _NotifyChanges(L"Has" #name, L#name);    \
         }                                            \
     }                                                \
-    auto name##OverrideSource() { return target.name##OverrideSource(); }
+    auto name##OverrideSource() { return target.name##OverrideSource().as<decltype(target)>(); }
```

Comment:
this seems inconspicuous, but also, wtf is it doing? A comment (below, outside the macro) might help enlighten as to why that's needed.